projects/packages/jetpack-mu-wpcom/src/features/wpcom-profile-settings/profile-settings-link-to-wpcom.php

```diff
@@ -41,19 +41,27 @@ function wpcom_profile_settings_add_links_to_wpcom() {
 		array(
 			'language'             => array(
 				'link' => esc_url( 'https://wordpress.com/me/account' ),
-				'text' => __( 'Your admin interface language is managed on WordPress.com Account settings', 'jetpack-mu-wpcom' ),
+				'text' => __( 'Manage your WordPress.com account language ↗', 'jetpack-mu-wpcom' ),
 			),
-			'synced'               => array(
+			'name'                 => array(
 				'link' => esc_url( 'https://wordpress.com/me' ),
-				'text' => __( 'You can manage your profile on WordPress.com Profile settings (First / Last / Display Names, Website, and Biographical Info)', 'jetpack-mu-wpcom' ),
+				'text' => __( 'Manage your WordPress.com profile ↗', 'jetpack-mu-wpcom' ),
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

Is using "↗" in Core the standard way to indicate an external link?

Reply:
I haven't seen any standard way in Core. It could be anything, I just wanted to "warn" the user that they are opening another domain, to minimize the effect of "unexpected Calypso jumping" surprise.

Reply:
Make sense. Maybe it's a good idea to use "↗" for other places linking to Calypso for consistency.

Reply:
The idea is to NOT link to Calypso at all from wp-admin screens, but this `profile.php` is a sad exception T_T

Reply:
> but this profile.php is a sad exception

Yes! I mean, if we had to do it somewhere by any chance 😄